debezium-connector-db2/src/main/java/io/debezium/connector/db2/Db2ConnectorConfig.java

```diff
@@ -279,14 +279,17 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
             SERVER_NAME,
             DATABASE_NAME,
             SNAPSHOT_MODE,
-            RelationalDatabaseConnectorConfig.SNAPSHOT_SELECT_STATEMENT_OVERRIDES_BY_TABLE,
-            HistorizedRelationalDatabaseConnectorConfig.DATABASE_HISTORY,
-            RelationalDatabaseConnectorConfig.TABLE_WHITELIST,
-            RelationalDatabaseConnectorConfig.TABLE_BLACKLIST,
-            RelationalDatabaseConnectorConfig.TABLE_IGNORE_BUILTIN,
-            RelationalDatabaseConnectorConfig.COLUMN_BLACKLIST,
-            RelationalDatabaseConnectorConfig.DECIMAL_HANDLING_MODE,
-            RelationalDatabaseConnectorConfig.TIME_PRECISION_MODE,
+            Db2ConnectorConfig.SNAPSHOT_SELECT_STATEMENT_OVERRIDES_BY_TABLE,
+            Db2ConnectorConfig.DATABASE_HISTORY,
+            Db2ConnectorConfig.TABLE_WHITELIST,
+            Db2ConnectorConfig.TABLE_INCLUDE_LIST,
+            Db2ConnectorConfig.TABLE_BLACKLIST,
+            Db2ConnectorConfig.TABLE_EXCLUDE_LIST,
+            Db2ConnectorConfig.TABLE_IGNORE_BUILTIN,
+            Db2ConnectorConfig.COLUMN_BLACKLIST,
+            Db2ConnectorConfig.COLUMN_EXCLUDE_LIST,
+            Db2ConnectorConfig.DECIMAL_HANDLING_MODE,
+            Db2ConnectorConfig.TIME_PRECISION_MODE,
```

Comment:
Beside adding the new include/exclude list properties I did some housekeeping here to move the parameters to the ConnectorConfig class related to that connector in order to local rewrites work and from architectural perspective we have less dependencies to "the outside".

Reply:
Will be reverted

Reply:
Done